evaporate.js

```diff
@@ -1847,10 +1800,46 @@
         return code.length ? {code: code, msg: msg} : {};
     }
 
+    var EvaporateHistoryCache = function () {
+        var result = false;
+
+        if (typeof window !== 'undefined') {
+            if ('localStorage' in window) {
+                try {
+                    localStorage.setItem('___test', 'OK');
+                    var test = localStorage.getItem('___test');
+                    localStorage.removeItem('___test');
+
+                    result = test === 'OK';
+                } catch (e) { }
+            }
+        }
+
+        if (result) {
+            this._cache = localStorage;
+        } else {
+            this._cache = {
```

Comment:
For browsers without localStorage or if localStorage is disabled. This feature was previously managed through the `supported` property.